src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'graphql-crunch' {
+  export function crunch(data: object | object[]): any[]
```

Comment:
Can’t data be just `any`? I’d also say the result is an array of (string, number, boolean, array of numbers or object with string keys and numeric values)

Reply:
I wasn't sure to know all the possible options for the result, but now yeah, we can definitely remove `any` from the result and replace `object` with `any`